CONTRIBUTING.md

```diff
@@ -1,195 +1,115 @@
+- [Accept the Contributor License Agreement](#accept-the-contributor-license-agreement)
+- [Propose changes to Ballerina](#propose-changes-to-ballerina)
 
-Import the Ballerina project to IntelliJ IDEA similar to any other Gradle project. 
+## Get started
 
-* Navigate to `File | Open` menu.
-* Browse the filesystem and Select Ballerina Project's root directory.
-   * If prompted to import Gradle project select `Auto import`.
-   * Set Project SDK as Java 1.8
+- Join the conversations at:
 
-##### Useful IDEA Plugins
+    - [StackOverflow](https://stackoverflow.com/questions/tagged/ballerina): to get help with Ballerina (use the Ballerina tag for any of your questions)
+    - [Slack](https://ballerina.io/community/slack/): for real-time discussions with the team and community
+    - [Ballerina-Dev Google Group](https://groups.google.com/forum/#!forum/ballerina-dev): (community mailing list) to discuss Ballerina roadmap, features, and related issues
+    - [Twitter](https://twitter.com/ballerinalang): to tweet about Ballerina (use the “#ballerinalang” hashtag) 
 
-* ANTLR 4 - https://plugins.jetbrains.com/plugin/7358-antlr-v4-grammar-plugin 
-* Ballerina - https://plugins.jetbrains.com/plugin/9520-ballerina (Need IDEA 2016.3 or newer)
+- Submitting a bug is just as important as contributing to code. Report an issue in the relevant repo out of the GitHub repos listed below. 
 
-#### Setup Eclipse
+    >**Tip:** If you are unsure whether you have found a bug, search existing issues in the corresponding repo on GitHub and raise it in the [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev).
+    - Compiler, runtime, standard library, or tooling: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo
+    - Language specification: <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo
+    - Website: <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo
+    - Security flaw: send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
 
-##### Importing a Ballerina Project
+-  Start with GitHub issues that can be fixed easily:
+    - Browse issues labeled "good first issue" in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
+    - Use comments on the issue itself to indicate that you will be working on it and get guidance and help.
 
-All Ballerina repositories are developed as Maven or Gradle projects. So you can import any Ballerina project to Eclipse similar to any Maven or Gradle project. 
+## Set up the prerequisites
+1. Download [Ballerina](https://ballerina.io) and go through the [learning resources](https://ballerina.io/learn).
+2. Read the <a href="https://ballerina.io/code-of-conduct">Ballerina Code of Conduct</a>.
```

Comment:
Shouldn't we move these two points to "Get Started". Nothing to setup in the given two points. Those are just readings.

Reply:
Added the 2 points under the ""Get Started" section and removed the "Set up the prerequisites" section.